orion/pkg/domains/achieve.go

```diff
@@ -18,6 +18,11 @@ type Achieve struct {
 	Message   string       `json:"message"`
 }
 
+type AchieveYearGroup struct {
+	Year         uint
```

Comment:
ah could you please include struct tags here `json:"year"` and for achievements below.
I want to make sure when these make it to the FE, they're lowercased.